modules/ROOT/pages/raise-error-component-reference.adoc

```diff
@@ -4,23 +4,90 @@ include::_attributes.adoc[]
+For core runtime errors types, you must use the implicit namespace and identifier, you can only customize the error's description message. For example:
+
+[source,xml,linenums]
+----
+<raise-error type="MULE:CONNECTIVITY" description="Error description message"/>
+----
+
+== Raise Custom Errors Types
+For custom error types, declare a new namespace. The namespace of an error type should help you identify the origin of an error. For example:
+
+[source,xml,linenums]
+----
+<raise-error type="CUSTOM:CUSTOM_ERROR" description="Error description message"/>
+----
+
 You cannot use existing namespaces from connectors or modules, since these have their defined namespaces.
-Once you declare a custom namespace by using it in a `raise-error`, you can use it in other
-`raise-error` components and custom types.
+Once you declare a custom namespace by using it in a `raise-error`, you can use it in other `raise-error` components and custom types.
+
+
+== Examples
+
+The following simple example flow raises a `VALUE: EXCEEDED_VALUE` error when the user enters a number greater than 10.
 
-The following example flow produces an `ACCOUNT:INSUFFICIENT_FUNDS` error when a
-transfer amount surpasses the available balance, preventing the transfer from
-taking place. Additionally, it generates a dynamic description by providing an
-expression with failure details.
+* The Choice router components adds conditional processing to the flow.
+* If the number value is greater than 10 in the request `http://localhost:8081/test?number=11`, the Raise Error component raises the error type `VALUE: EXCEEDED_VALUE` and description `Parameter value exceeded`.
+* Then, a Logger component logs the error message `The number value exceeds 10` in the On Error Continue component flow whose type is `VALUE: EXCEEDED_VALUE`.
+* If the number value is lower than 10 in the request `http://localhost:8081/test?number=1`, a Logger component returns the default message `The number value is lower than 10`.
+* If no number value parameter is passed in the request `http://localhost:8081/test`, a Logger component returns the message `The parameter number is missing`.
 
-.Example: XML Configuration of the Flow
+XML Configuration of the Flow:
+[source,xml,linenums]
+----
+<http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" doc:id="8e14c3c2-94ee-4394-b56f-f861fdb518fc" >
+  <http:listener-connection host="0.0.0.0" port="8081" />
+</http:listener-config>
+<flow name="raiseErrorFlow" doc:id="393d25ff-c699-42ad-a90c-653801b14b9f" >
+  <http:listener doc:name="Listener" doc:id="8c0f7286-0428-4578-b90e-bc1a8ae0ff03" config-ref="HTTP_Listener_config" path="/test"/>
+  <choice doc:name="Choice" doc:id="560d7e31-c122-4a95-9716-a9988a1c4a85" >
+    <when expression="#[message.attributes.queryParams.number &gt; 10]">
+      <raise-error doc:name="Raise error" doc:id="9cc4cf6c-5639-4151-8acc-d553a47ae4cb" type="VALUE: EXCEEDED_VALUE" description="Parameter value exceeded." />
```

Comment:
I don't think this is a good example. `VALUE` is too abstract to be a useful namespace, definitely not what we want people to get from this. What if instead of `number` you name the query parameter `age` and treat this as a validation some websites use to guarantee the user is over a certain age (I'm thinking a driving lessons platform which checks you are over 16)? Then the namespace could be `PRECONDITIONS`.